src/components/Hero/Hero.css

```diff
@@ -0,0 +1,146 @@
+/*Button is not working from react-bootstrap*/
+
+HTML {
```

Comment:
do you want to target everything ? can we use smth like the *  selector I have never seen this HTML getting used